2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,8 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  return Object.fromEntries(countryCurrencyCodes);
```

Comment:
You definitely took the easy one on this one!

If you had to manually do it without helper functions this would be the best way.
` let currencyCodeLookup = {};

  for (let countryCurrencyCode of countryCurrencyCodes) {
    let countryCode = countryCurrencyCode[0];
    let currencyCode = countryCurrencyCode[1];
    currencyCodeLookup[countryCode] = currencyCode;
  };`